src/components/auth/sign-in-card.tsx

```
@@ -43,15 +43,16 @@
-                Login with Github
+                {"Login with Github"}
               </Button>
             </div>
           </div>
         </CardContent>
       </Card>
       <div className="text-muted-foreground *:[a]:hover:text-primary text-center text-xs text-balance *:[a]:underline *:[a]:underline-offset-4">
-        By clicking continue, you agree to our <a href="#">Terms of Service</a>{" "}
-        and <a href="#">Privacy Policy</a>.
+        {"By clicking continue, you agree to our"}{" "}
+        <a href="#">{"Terms of Service"}</a> {"and"}{" "}
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)